### PR TITLE
Fix foreman-proxy logrotate with systemd

### PR DESCRIPTION
**Pull request by ekohl**
Untested but inspired by https://github.com/theforeman/foreman-packaging/pull/2622

### CODE REVIEW

File: debian/stretch/foreman-proxy/logrotate [hunk 1]
```diff
@@ -9,6 +9,10 @@
   compress
 	daily
   postrotate
-    [ -e /etc/init.d/foreman-proxy ] && /etc/init.d/foreman-proxy logrotate >/dev/null 2>&1 || true
+    if [ -d /run/systemd/system ] ; then
+      /bin/systemctl kill --signal=SIGUSR1 foreman-proxy >/dev/null 2>&1 || true
+    elif [ -e /etc/init.d/foreman-proxy ]
```
Comment (mmoll) [inline debian/stretch/foreman-proxy/logrotate]: `; then` is also needed here

**Comment by mmoll** [comment]
one small thing, looking good otherwise

**Comment by mmoll** [comment]
bionic also needs this change now

**Comment by ekohl** [comment]
Updated

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**